lib/improver/wxcode/weather_symbols.py

```diff
@@ -164,7 +191,10 @@ def check_input_cubes(self, cubes):
             for item in missing_data:
                 msg = msg + dyn_msg.format(*item)
             raise IOError(msg)
-        return
+
+        if len(optional_node_data_missing) == 0:
```

Comment:
`if not optional_node_data_missing`

Reply:
I have changed this. I'm assuming this runs quicker because it makes less sense.

Reply:
Not quicker, no - it's just a common python idiom to check to see if a collection is empty/not.

Reply:
It might be more pythonic but from a logically point of view I think it is less understandable, still I've changed it now.